implementation-status.md

```diff
@@ -32,13 +30,18 @@ Tip: Chrome channel releases are tracked at [https://googlechromelabs.github.io/
 * [Issue 854935](http://crbug.com/854935): Closed captions in Picture-in-Picture are not supported.
 
 # Microsoft Edge
-https://dev.windows.com/en-us/microsoft-edge/platform/status/pictureinpicture
+Under consideration.
+https://developer.microsoft.com/en-us/microsoft-edge/platform/status/pictureinpicture/
 
 # Safari
+No public signals. 
 https://bugs.webkit.org/show_bug.cgi?id=182688
 
 # Firefox
+No public signals. 
```

Comment:
I would say there was a lot of public signals ^_^

Reply:
Heh, our official(tm) position is to “defer”:
https://github.com/mozilla/standards-positions/issues/72

 That doesn’t mean we wont do it, just not right now.

Reply:
FYI I've opened https://github.com/WICG/picture-in-picture/pull/155

Reply:
So, Mozilla is happy for you all the figure out the spec details, tests, etc. Can you please ping me when you think the spec is CR ready and it’s in Safari Tech Preview? I think then might be a good time to re-evaluate adding the API. Given that we have PiP working, adding the DOM bits should be fairly trivial (famous last words).